lnwallet/reservation.go

```diff
@@ -521,6 +524,16 @@ func (r *ChannelReservation) FundingOutpoint() *wire.OutPoint {
 	return &r.partialState.FundingOutpoint
 }
 
+// SetOurUpfrontShutdown sets the upfront shutdown address on our contribution
+// and partial state in the reservation.
+func (r *ChannelReservation) SetOurUpfrontShutdown(shutdown lnwire.DeliveryAddress) {
+	r.Lock()
+	defer r.Unlock()
+
+	r.ourContribution.UpfrontShutdown = shutdown
+	r.partialState.LocalShutdownScript = shutdown
```

Comment:
is this necessary to set here?

Reply:
We  use the values on `partialState` when we call `SyncPending` in `handleFundingCounterPartySigs` or `handleSingleFunderSigs`, which both use the `ChannelReservation.partialState`. 

I'm going to update to only set the values on `partialState` when we write to disk, but I do think it's a bit strange to have these two out of sync with each other. Fine for now, but future changes are going to have to know that this value is not set on `partialState`.